service/query/mini_test.go

```diff
@@ -214,6 +217,24 @@ func TestParse(t *testing.T) {
 			"CREATE index",
 			[]qp.Table{qp.Table{Db: "brannigan", Table: "percona"}},
 		},
+		// PMM-1892. Upgraded Vitess libraries to support this query.
+		// Notice that the query below is not exactly the same reported in the ticket; this
+		// query has `auto_increment` between backticks because it is a reserved MySQL word
+		// but MySQL accepts it anyway as a field name while Vitess doesn't.
+		example{
+			"SELECT table_schema, table_name, column_name, `auto_increment`, " +
```

Comment:
This seems to be `mysqld_exporter` query, so I've escaped `auto_increment` https://github.com/percona/mysqld_exporter/pull/17/files#diff-a720d3562307aca7a841eea9ca784a57R12